packages/@lwc/engine-core/src/framework/modules/props.ts

```diff
@@ -43,6 +43,20 @@ export function patchProps(
             isFirstPatch ||
             cur !== (isLiveBindingProp(sel, key) ? getProperty(elm!, key) : oldProps[key])
         ) {
+            // Additional verification if properties are supported by the element
```

Comment:
Moved this validation from [@lwc/engine-dom/src/renderer.ts](https://github.com/salesforce/lwc/pull/2997/files#diff-d700e3ea8d9babd8109d1b3e94e40cf4f45231cfd3ab2a47b760ca7dfff3d3bc)